report_py3o/models/py3o_report.py

```diff
@@ -181,40 +208,87 @@ def create_single_pdf(self, cr, uid, ids, data, report_xml, context=None):
                 report_xml.py3o_server_id.url, data=fields, files=files)
             if r.status_code != 200:
                 # server says we have an issue... let's tell that to enduser
-                raise exceptions.Warning(
+                raise UserError(
                     _('Fusion server error %s') % r.text,
                 )
 
             # Here is a little joke about Odoo
             # we do nice chunked reading from the network...
             chunk_size = 1024
             with NamedTemporaryFile(
-                    suffix=filetype,
-                    prefix='py3o-template-'
+                suffix=filetype,
+                prefix='py3o-template-'
             ) as fd:
                 for chunk in r.iter_content(chunk_size):
                     fd.write(chunk)
                 fd.seek(0)
                 # ... but odoo wants the whole data in memory anyways :)
                 res = fd.read()
+        self._postprocess_report(
+            res, model_instance.id, save_in_attachment)
+        return res, "." + self.ir_actions_report_xml_id.py3o_filetype
```

Comment:
I fear you re-introduced the "double dot in filename" bug, that I fixed with this commit https://github.com/akretion/reporting-engine/commit/b702d15a2982f0075c1cacd7a0c66b2138372866

Did you test it this PR and check the filenames of the py3o reports generated ?